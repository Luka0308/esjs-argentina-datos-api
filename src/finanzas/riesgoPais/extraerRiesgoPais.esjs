importar axios desde 'axios'
importar { format, subDays, addDays, isBefore como esAntes, parse } desde 'date-fns'
importar { escribirRuta, leerRuta } desde '@/utils/rutas.esjs'

exportar asincrono funcion extraerRiesgoPais(desde, hasta) {
  intentar {
    const respuesta = esperar axios.get(
      `https://mercados.ambito.com/riesgopais/historico-general/${format(desde, 'yyyy-MM-dd')}/${format(hasta, 'yyyy-MM-dd')}`,
    )

    const items = []

    para (const item de respuesta.data) {
      si (item[0] === 'Fecha') {
        continuar
      }

      items.agregar({
        valor: interpretarDecimal(item[1]),
        fecha: format(parse(item[0], 'dd-MM-yyyy', crear Fecha()), 'yyyy-MM-dd'),
      })
    }

    retornar items
  } capturar (error) {
    consola.error(error)
    retornar []
  }
}
