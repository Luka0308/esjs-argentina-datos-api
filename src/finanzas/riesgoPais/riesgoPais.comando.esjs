importar { format, subDays, addDays, isBefore como esAntes, parse } desde 'date-fns'
importar { escribirRuta, leerRuta } desde '@/utils/rutas.esjs'
importar { extraerRiesgoPais } desde '@/finanzas/riesgoPais/extraerRiesgoPais.esjs'

exportar porDefecto asincrono funcion () {
  const riesgoPaisValor = esperar extraerRiesgoPais(
    subDays(crear Fecha(), 10),
    crear Fecha(),
  )

  const valoresRellenados = esperar rellenarValores(riesgoPaisValor.reverso())

  const ultimoValor = valoresRellenados[valoresRellenados.longitud - 1]

  const rutasGuardadas = [
    esperar guardarUltimo(ultimoValor),
    esperar guardarHistorico(valoresRellenados),
  ]

  retornar rutasGuardadas
}

asincrono funcion rellenarValores(valores) {
  const primerValor = valores[0]

  const nuevosValores = [primerValor]

  const fechaInicial = parse(primerValor.fecha, 'yyyy-MM-dd', crear Fecha())
  mut desde = subDays(fechaInicial, 0)
  const hasta = addDays(crear Fecha(), 0)

  mientras (esAntes(desde, hasta)) {
    const fecha = format(desde, 'yyyy-MM-dd')

    const valor = valores.buscar(v => v.fecha === fecha)

    si (valor) {
      nuevosValores.agregar(valor)
    } sino {
      const valorAnterior = nuevosValores[nuevosValores.longitud - 1]

      nuevosValores.agregar({
        valor: valorAnterior.valor,
        fecha,
      })
    }

    desde = addDays(desde, 1)
  }

  retornar nuevosValores
}

asincrono funcion guardarUltimo(ultimoRiesgoPais, fecha) {
  const ruta = '/finanzas/indices/riesgo-pais/ultimo'

  esperar escribirRuta(ruta, ultimoRiesgoPais)

  retornar ruta
}

asincrono funcion guardarHistorico(valores) {
  const ruta = '/finanzas/indices/riesgo-pais'

  const historico = esperar leerRuta(ruta)

  const nuevoHistorico = historico.mapear(valor => {
    si (valores.algun(v => v.fecha === valor.fecha)) {
      const nuevoValor = valores.buscar(v => v.fecha === valor.fecha)

      retornar {
        valor: nuevoValor.valor,
        fecha: valor.fecha,
      }
    }

    retornar valor
  })

  esperar escribirRuta(ruta, nuevoHistorico)

  retornar ruta
}
