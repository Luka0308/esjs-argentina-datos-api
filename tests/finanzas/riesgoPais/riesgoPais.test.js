
import { describe, expect, it } from 'vitest'
import { extraerRiesgoPais } from '@/finanzas/riesgoPais/extraerRiesgoPais.esjs'
import riesgoPais from '@/finanzas/riesgoPais/riesgoPais.comando.esjs'

describe('riesgoPais', () => {
  it('extraer valores', async () => {
    const riesgoPais = await extraerRiesgoPais(
      new Date('2024-01-01'),
      new Date('2024-12-31')
    )

    expect(riesgoPais).toMatchSnapshot()
  })

  it('comando', async () => {
    const rutasGuardadas = await riesgoPais()

    expect(rutasGuardadas).toMatchSnapshot()
  })
})
