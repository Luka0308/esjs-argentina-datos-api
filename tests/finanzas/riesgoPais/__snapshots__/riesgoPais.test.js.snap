// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`riesgoPais > comando 1`] = `
[
  "/finanzas/indices/riesgo-pais/ultimo",
  "/finanzas/indices/riesgo-pais",
]
`;

exports[`riesgoPais > extraer valores 1`] = `
[
  {
    "fecha": "2024-12-27",
    "valor": 627,
  },
  {
    "fecha": "2024-12-26",
    "valor": 631,
  },
  {
    "fecha": "2024-12-25",
    "valor": 650,
  },
  {
    "fecha": "2024-12-24",
    "valor": 649,
  },
  {
    "fecha": "2024-12-23",
    "valor": 649,
  },
  {
    "fecha": "2024-12-20",
    "valor": 671,
  },
  {
    "fecha": "2024-12-19",
    "valor": 663,
  },
  {
    "fecha": "2024-12-18",
    "valor": 658,
  },
  {
    "fecha": "2024-12-17",
    "valor": 673,
  },
  {
    "fecha": "2024-12-16",
    "valor": 677,
  },
  {
    "fecha": "2024-12-13",
    "valor": 708,
  },
  {
    "fecha": "2024-12-12",
    "valor": 715,
  },
  {
    "fecha": "2024-12-11",
    "valor": 719,
  },
  {
    "fecha": "2024-12-10",
    "valor": 735,
  },
  {
    "fecha": "2024-12-09",
    "valor": 737,
  },
  {
    "fecha": "2024-12-06",
    "valor": 769,
  },
  {
    "fecha": "2024-12-05",
    "valor": 760,
  },
  {
    "fecha": "2024-12-04",
    "valor": 751,
  },
  {
    "fecha": "2024-12-03",
    "valor": 754,
  },
  {
    "fecha": "2024-12-02",
    "valor": 749,
  },
  {
    "fecha": "2024-11-29",
    "valor": 752,
  },
  {
    "fecha": "2024-11-28",
    "valor": 752,
  },
  {
    "fecha": "2024-11-27",
    "valor": 755,
  },
  {
    "fecha": "2024-11-26",
    "valor": 751,
  },
  {
    "fecha": "2024-11-25",
    "valor": 748,
  },
  {
    "fecha": "2024-11-22",
    "valor": 745,
  },
  {
    "fecha": "2024-11-21",
    "valor": 734,
  },
  {
    "fecha": "2024-11-20",
    "valor": 720,
  },
  {
    "fecha": "2024-11-19",
    "valor": 746,
  },
  {
    "fecha": "2024-11-18",
    "valor": 769,
  },
  {
    "fecha": "2024-11-15",
    "valor": 769,
  },
  {
    "fecha": "2024-11-14",
    "valor": 772,
  },
  {
    "fecha": "2024-11-13",
    "valor": 800,
  },
  {
    "fecha": "2024-11-12",
    "valor": 819,
  },
  {
    "fecha": "2024-11-11",
    "valor": 850,
  },
  {
    "fecha": "2024-11-08",
    "valor": 850,
  },
  {
    "fecha": "2024-11-07",
    "valor": 859,
  },
  {
    "fecha": "2024-11-06",
    "valor": 870,
  },
  {
    "fecha": "2024-11-05",
    "valor": 933,
  },
  {
    "fecha": "2024-11-04",
    "valor": 955,
  },
  {
    "fecha": "2024-11-01",
    "valor": 955,
  },
  {
    "fecha": "2024-10-31",
    "valor": 984,
  },
  {
    "fecha": "2024-10-30",
    "valor": 956,
  },
  {
    "fecha": "2024-10-29",
    "valor": 933,
  },
  {
    "fecha": "2024-10-28",
    "valor": 914,
  },
  {
    "fecha": "2024-10-25",
    "valor": 967,
  },
  {
    "fecha": "2024-10-24",
    "valor": 1036,
  },
  {
    "fecha": "2024-10-23",
    "valor": 1103,
  },
  {
    "fecha": "2024-10-22",
    "valor": 1099,
  },
  {
    "fecha": "2024-10-21",
    "valor": 1100,
  },
  {
    "fecha": "2024-10-18",
    "valor": 1097,
  },
  {
    "fecha": "2024-10-17",
    "valor": 1099,
  },
  {
    "fecha": "2024-10-16",
    "valor": 1052,
  },
  {
    "fecha": "2024-10-15",
    "valor": 1044,
  },
  {
    "fecha": "2024-10-14",
    "valor": 1106,
  },
  {
    "fecha": "2024-10-11",
    "valor": 1106,
  },
  {
    "fecha": "2024-10-10",
    "valor": 1126,
  },
  {
    "fecha": "2024-10-09",
    "valor": 1121,
  },
  {
    "fecha": "2024-10-08",
    "valor": 1188,
  },
  {
    "fecha": "2024-10-07",
    "valor": 1207,
  },
  {
    "fecha": "2024-10-04",
    "valor": 1258,
  },
  {
    "fecha": "2024-10-03",
    "valor": 1265,
  },
  {
    "fecha": "2024-10-02",
    "valor": 1276,
  },
  {
    "fecha": "2024-10-01",
    "valor": 1288,
  },
  {
    "fecha": "2024-09-30",
    "valor": 1290,
  },
  {
    "fecha": "2024-09-27",
    "valor": 1311,
  },
  {
    "fecha": "2024-09-26",
    "valor": 1302,
  },
  {
    "fecha": "2024-09-25",
    "valor": 1304,
  },
  {
    "fecha": "2024-09-24",
    "valor": 1303,
  },
  {
    "fecha": "2024-09-23",
    "valor": 1308,
  },
  {
    "fecha": "2024-09-20",
    "valor": 1319,
  },
  {
    "fecha": "2024-09-19",
    "valor": 1301,
  },
  {
    "fecha": "2024-09-18",
    "valor": 1363,
  },
  {
    "fecha": "2024-09-17",
    "valor": 1370,
  },
  {
    "fecha": "2024-09-16",
    "valor": 1365,
  },
  {
    "fecha": "2024-09-13",
    "valor": 1400,
  },
  {
    "fecha": "2024-09-12",
    "valor": 1422,
  },
  {
    "fecha": "2024-09-11",
    "valor": 1445,
  },
  {
    "fecha": "2024-09-10",
    "valor": 1445,
  },
  {
    "fecha": "2024-09-09",
    "valor": 1442,
  },
  {
    "fecha": "2024-09-06",
    "valor": 1483,
  },
  {
    "fecha": "2024-09-05",
    "valor": 1452,
  },
  {
    "fecha": "2024-09-04",
    "valor": 1465,
  },
  {
    "fecha": "2024-09-03",
    "valor": 1467,
  },
  {
    "fecha": "2024-09-02",
    "valor": 1433,
  },
  {
    "fecha": "2024-08-30",
    "valor": 1433,
  },
  {
    "fecha": "2024-08-29",
    "valor": 1444,
  },
  {
    "fecha": "2024-08-28",
    "valor": 1484,
  },
  {
    "fecha": "2024-08-27",
    "valor": 1510,
  },
  {
    "fecha": "2024-08-26",
    "valor": 1516,
  },
  {
    "fecha": "2024-08-23",
    "valor": 1516,
  },
  {
    "fecha": "2024-08-22",
    "valor": 1560,
  },
  {
    "fecha": "2024-08-21",
    "valor": 1525,
  },
  {
    "fecha": "2024-08-20",
    "valor": 1493,
  },
  {
    "fecha": "2024-08-19",
    "valor": 1471,
  },
  {
    "fecha": "2024-08-16",
    "valor": 1471,
  },
  {
    "fecha": "2024-08-15",
    "valor": 1490,
  },
  {
    "fecha": "2024-08-14",
    "valor": 1515,
  },
  {
    "fecha": "2024-08-13",
    "valor": 1557,
  },
  {
    "fecha": "2024-08-12",
    "valor": 1560,
  },
  {
    "fecha": "2024-08-09",
    "valor": 1558,
  },
  {
    "fecha": "2024-08-08",
    "valor": 1552,
  },
  {
    "fecha": "2024-08-07",
    "valor": 1574,
  },
  {
    "fecha": "2024-08-06",
    "valor": 1625,
  },
  {
    "fecha": "2024-08-05",
    "valor": 1653,
  },
  {
    "fecha": "2024-08-02",
    "valor": 1612,
  },
  {
    "fecha": "2024-08-01",
    "valor": 1550,
  },
  {
    "fecha": "2024-07-31",
    "valor": 1507,
  },
  {
    "fecha": "2024-07-30",
    "valor": 1556,
  },
  {
    "fecha": "2024-07-29",
    "valor": 1553,
  },
  {
    "fecha": "2024-07-26",
    "valor": 1562,
  },
  {
    "fecha": "2024-07-25",
    "valor": 1589,
  },
  {
    "fecha": "2024-07-24",
    "valor": 1577,
  },
  {
    "fecha": "2024-07-23",
    "valor": 1551,
  },
  {
    "fecha": "2024-07-22",
    "valor": 1570,
  },
  {
    "fecha": "2024-07-19",
    "valor": 1590,
  },
  {
    "fecha": "2024-07-18",
    "valor": 1587,
  },
  {
    "fecha": "2024-07-17",
    "valor": 1593,
  },
  {
    "fecha": "2024-07-16",
    "valor": 1584,
  },
  {
    "fecha": "2024-07-15",
    "valor": 1555,
  },
  {
    "fecha": "2024-07-12",
    "valor": 1511,
  },
  {
    "fecha": "2024-07-11",
    "valor": 1473,
  },
  {
    "fecha": "2024-07-10",
    "valor": 1477,
  },
  {
    "fecha": "2024-07-09",
    "valor": 1491,
  },
  {
    "fecha": "2024-07-08",
    "valor": 1496,
  },
  {
    "fecha": "2024-07-05",
    "valor": 1406,
  },
  {
    "fecha": "2024-07-04",
    "valor": 1523,
  },
  {
    "fecha": "2024-07-03",
    "valor": 1521,
  },
  {
    "fecha": "2024-07-02",
    "valor": 1536,
  },
  {
    "fecha": "2024-07-01",
    "valor": 1513,
  },
  {
    "fecha": "2024-06-28",
    "valor": 1456,
  },
  {
    "fecha": "2024-06-27",
    "valor": 1445,
  },
  {
    "fecha": "2024-06-26",
    "valor": 1429,
  },
  {
    "fecha": "2024-06-25",
    "valor": 1440,
  },
  {
    "fecha": "2024-06-24",
    "valor": 1443,
  },
  {
    "fecha": "2024-06-21",
    "valor": 1420,
  },
  {
    "fecha": "2024-06-20",
    "valor": 1402,
  },
  {
    "fecha": "2024-06-19",
    "valor": 1384,
  },
  {
    "fecha": "2024-06-18",
    "valor": 1385,
  },
  {
    "fecha": "2024-06-17",
    "valor": 1382,
  },
  {
    "fecha": "2024-06-14",
    "valor": 1382,
  },
  {
    "fecha": "2024-06-13",
    "valor": 1421,
  },
  {
    "fecha": "2024-06-12",
    "valor": 1484,
  },
  {
    "fecha": "2024-06-11",
    "valor": 1525,
  },
  {
    "fecha": "2024-06-10",
    "valor": 1488,
  },
  {
    "fecha": "2024-06-07",
    "valor": 1582,
  },
  {
    "fecha": "2024-06-06",
    "valor": 1596,
  },
  {
    "fecha": "2024-06-05",
    "valor": 1528,
  },
  {
    "fecha": "2024-06-04",
    "valor": 1494,
  },
  {
    "fecha": "2024-06-03",
    "valor": 1414,
  },
  {
    "fecha": "2024-05-31",
    "valor": 1312,
  },
  {
    "fecha": "2024-05-30",
    "valor": 1312,
  },
  {
    "fecha": "2024-05-29",
    "valor": 1376,
  },
  {
    "fecha": "2024-05-28",
    "valor": 1416,
  },
  {
    "fecha": "2024-05-27",
    "valor": 1443,
  },
  {
    "fecha": "2024-05-24",
    "valor": 1443,
  },
  {
    "fecha": "2024-05-23",
    "valor": 1433,
  },
  {
    "fecha": "2024-05-22",
    "valor": 1394,
  },
  {
    "fecha": "2024-05-21",
    "valor": 1298,
  },
  {
    "fecha": "2024-05-20",
    "valor": 1258,
  },
  {
    "fecha": "2024-05-17",
    "valor": 1263,
  },
  {
    "fecha": "2024-05-16",
    "valor": 1254,
  },
  {
    "fecha": "2024-05-15",
    "valor": 1235,
  },
  {
    "fecha": "2024-05-14",
    "valor": 1244,
  },
  {
    "fecha": "2024-05-13",
    "valor": 1248,
  },
  {
    "fecha": "2024-05-10",
    "valor": 1272,
  },
  {
    "fecha": "2024-05-09",
    "valor": 1263,
  },
  {
    "fecha": "2024-05-08",
    "valor": 1234,
  },
  {
    "fecha": "2024-05-07",
    "valor": 1219,
  },
  {
    "fecha": "2024-05-06",
    "valor": 1229,
  },
  {
    "fecha": "2024-05-03",
    "valor": 1243,
  },
  {
    "fecha": "2024-05-02",
    "valor": 1236,
  },
  {
    "fecha": "2024-05-01",
    "valor": 1224,
  },
  {
    "fecha": "2024-04-30",
    "valor": 1216,
  },
  {
    "fecha": "2024-04-29",
    "valor": 1208,
  },
  {
    "fecha": "2024-04-26",
    "valor": 1210,
  },
  {
    "fecha": "2024-04-25",
    "valor": 1234,
  },
  {
    "fecha": "2024-04-24",
    "valor": 1205,
  },
  {
    "fecha": "2024-04-23",
    "valor": 1174,
  },
  {
    "fecha": "2024-04-22",
    "valor": 1148,
  },
  {
    "fecha": "2024-04-19",
    "valor": 1216,
  },
  {
    "fecha": "2024-04-18",
    "valor": 1242,
  },
  {
    "fecha": "2024-04-17",
    "valor": 1270,
  },
  {
    "fecha": "2024-04-16",
    "valor": 1381,
  },
  {
    "fecha": "2024-04-15",
    "valor": 1375,
  },
  {
    "fecha": "2024-04-12",
    "valor": 1331,
  },
  {
    "fecha": "2024-04-11",
    "valor": 1290,
  },
  {
    "fecha": "2024-04-10",
    "valor": 1276,
  },
  {
    "fecha": "2024-04-09",
    "valor": 1220,
  },
  {
    "fecha": "2024-04-08",
    "valor": 1205,
  },
  {
    "fecha": "2024-04-05",
    "valor": 1302,
  },
  {
    "fecha": "2024-04-04",
    "valor": 1359,
  },
  {
    "fecha": "2024-04-03",
    "valor": 1419,
  },
  {
    "fecha": "2024-04-02",
    "valor": 1439,
  },
  {
    "fecha": "2024-04-01",
    "valor": 1439,
  },
  {
    "fecha": "2024-03-29",
    "valor": 1439,
  },
  {
    "fecha": "2024-03-28",
    "valor": 1439,
  },
  {
    "fecha": "2024-03-27",
    "valor": 1442,
  },
  {
    "fecha": "2024-03-26",
    "valor": 1430,
  },
  {
    "fecha": "2024-03-25",
    "valor": 1421,
  },
  {
    "fecha": "2024-03-22",
    "valor": 1432,
  },
  {
    "fecha": "2024-03-21",
    "valor": 1460,
  },
  {
    "fecha": "2024-03-20",
    "valor": 1498,
  },
  {
    "fecha": "2024-03-19",
    "valor": 1535,
  },
  {
    "fecha": "2024-03-18",
    "valor": 1581,
  },
  {
    "fecha": "2024-03-15",
    "valor": 1603,
  },
  {
    "fecha": "2024-03-14",
    "valor": 1648,
  },
  {
    "fecha": "2024-03-13",
    "valor": 1596,
  },
  {
    "fecha": "2024-03-12",
    "valor": 1645,
  },
  {
    "fecha": "2024-03-11",
    "valor": 1713,
  },
  {
    "fecha": "2024-03-08",
    "valor": 1690,
  },
  {
    "fecha": "2024-03-07",
    "valor": 1643,
  },
  {
    "fecha": "2024-03-06",
    "valor": 1621,
  },
  {
    "fecha": "2024-03-05",
    "valor": 1617,
  },
  {
    "fecha": "2024-03-04",
    "valor": 1590,
  },
  {
    "fecha": "2024-03-01",
    "valor": 1635,
  },
  {
    "fecha": "2024-02-29",
    "valor": 1705,
  },
  {
    "fecha": "2024-02-28",
    "valor": 1730,
  },
  {
    "fecha": "2024-02-27",
    "valor": 1753,
  },
  {
    "fecha": "2024-02-26",
    "valor": 1709,
  },
  {
    "fecha": "2024-02-23",
    "valor": 1686,
  },
  {
    "fecha": "2024-02-22",
    "valor": 1710,
  },
  {
    "fecha": "2024-02-21",
    "valor": 1786,
  },
  {
    "fecha": "2024-02-20",
    "valor": 1762,
  },
  {
    "fecha": "2024-02-19",
    "valor": 1788,
  },
  {
    "fecha": "2024-02-16",
    "valor": 1788,
  },
  {
    "fecha": "2024-02-15",
    "valor": 1895,
  },
  {
    "fecha": "2024-02-14",
    "valor": 1930,
  },
  {
    "fecha": "2024-02-13",
    "valor": 2032,
  },
  {
    "fecha": "2024-02-12",
    "valor": 1993,
  },
  {
    "fecha": "2024-02-09",
    "valor": 1993,
  },
  {
    "fecha": "2024-02-08",
    "valor": 1956,
  },
  {
    "fecha": "2024-02-07",
    "valor": 1928,
  },
  {
    "fecha": "2024-02-06",
    "valor": 1832,
  },
  {
    "fecha": "2024-02-05",
    "valor": 1869,
  },
  {
    "fecha": "2024-02-02",
    "valor": 1919,
  },
  {
    "fecha": "2024-02-01",
    "valor": 1959,
  },
  {
    "fecha": "2024-01-31",
    "valor": 1950,
  },
  {
    "fecha": "2024-01-30",
    "valor": 1936,
  },
  {
    "fecha": "2024-01-29",
    "valor": 1888,
  },
  {
    "fecha": "2024-01-26",
    "valor": 1856,
  },
  {
    "fecha": "2024-01-25",
    "valor": 1870,
  },
  {
    "fecha": "2024-01-24",
    "valor": 1897,
  },
  {
    "fecha": "2024-01-23",
    "valor": 1892,
  },
  {
    "fecha": "2024-01-22",
    "valor": 1935,
  },
  {
    "fecha": "2024-01-19",
    "valor": 1971,
  },
  {
    "fecha": "2024-01-18",
    "valor": 1966,
  },
  {
    "fecha": "2024-01-17",
    "valor": 1948,
  },
  {
    "fecha": "2024-01-16",
    "valor": 1920,
  },
  {
    "fecha": "2024-01-15",
    "valor": 1905,
  },
  {
    "fecha": "2024-01-12",
    "valor": 1904,
  },
  {
    "fecha": "2024-01-11",
    "valor": 1926,
  },
  {
    "fecha": "2024-01-10",
    "valor": 1985,
  },
  {
    "fecha": "2024-01-09",
    "valor": 2102,
  },
  {
    "fecha": "2024-01-08",
    "valor": 2049,
  },
  {
    "fecha": "2024-01-05",
    "valor": 1998,
  },
  {
    "fecha": "2024-01-04",
    "valor": 2018,
  },
  {
    "fecha": "2024-01-03",
    "valor": 1986,
  },
  {
    "fecha": "2024-01-02",
    "valor": 1938,
  },
  {
    "fecha": "2024-01-01",
    "valor": 1906,
  },
]
`;
